### ISSUE TITLE
Emphasize current week in list of weeks that are due

### ISSUE THREAD

**Issue by kingcomma** [18F/tock#1770]
As someone who tocks their time, it can be difficult to find the _current_ week in the list of weeks that are due on the Tock homepage. The visual differences between the links are usually subtle (June ..., June ..., June ...) and the current week is at the bottom of the list when my instinct is for it to be at the top. This has led to me filling out the wrong week and then having to request a tock admin's time to perform a correction.

At the risk of designing as a non-designer, some way to visually indicate the current week (if it has not yet been completed) in the list would be helpful. For example, the list currently looks like:

<img width="161" alt="Screenshot 2024-06-21 at 4 05 59 PM" src="https://github.com/18F/tock/assets/3246955/684911d9-c3cd-410c-8037-eaceb93ce5e7">

Compare to this re-ordered list with the current week bolded:

<img width="191" alt="Screenshot 2024-06-21 at 4 07 41 PM" src="https://github.com/18F/tock/assets/3246955/ddc08a73-c11d-4f8c-a099-0423ec5a78bd">

**Comment by juliaklindpaintner** [comment]
Was here to submit this same issue, so adding a heavy plus 1 instead. This same thing has happened to 2 of my direct reports in the last 2 weeks!

**Comment by melchoyce** [comment]
Quick suggestion. What if we did two things:

1. Call out the current week (and past due) separately from future weeks
2. Reverse how the weeks are displayed so the sooner ones at the top and later ones at the bottom

![image](https://github.com/user-attachments/assets/8d479371-f868-439c-a42a-a23ca4882dea)

**Comment by caleywoods** [comment]
> Quick suggestion. What if we did two things:
> 
> 1. Call out the current week (and past due) separately from future weeks
> 2. Reverse how the weeks are displayed so the sooner ones at the top and later ones at the bottom
> 
> ![image](https://github.com/user-attachments/assets/8d479371-f868-439c-a42a-a23ca4882dea)
> 
> If this sounds good, we'll need an engineer who can implement it.

Yes! I was reading this issue and looking at the original idea and thinking this exact same thing.

**Comment by peterrowland** [comment]
I have a draft PR open for this that only reverses the sort order and bolds the current week. While it would be possible to implement the design with a separate list for the current week, it doesn't account for overdue items which should not be in the current week, and would be confusing if they appeared underneath it in the 'weeks ending' list. 

I attempted to create a version with an additional list for overdue items if there are any, but realized that this requires more conditional logic to make lists visible and append items to them than I think can be accomplished with the inline scripting language. I may be wrong about this, but I think it would need a redesign.

The two options I see without doing that are use this simplified version that I've made, or create a separate list for the current week and accept that overdue items will appear out of order.
